gallery/src/pages/AccessibilityChecklist.tsx

```diff
@@ -0,0 +1,1516 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+import {
+  NxP,
+  NxH3,
+  NxH4,
+  NxList,
+  NxTextLink,
+  NxCheckbox,
+  NxStatefulCheckbox,
+  NxFieldset,
+  useToggle
+} from '@sonatype/react-shared-components';
+
+function Accessibility() {
+
+  const [isA, toggleA] = useToggle(false),
+      [isAA, toggleAA] = useToggle(false),
+      [isAAA, toggleAAA] = useToggle(false),
+      [isReco, toggleReco] = useToggle(false);
+
+  return (
+    <GalleryTile title="WCAG Accessibility UX Checklist" className="nx-tile--accessibility-list">
+      <NxP>
+        Accessibility UX checklist to serve as a method for designers and developers to assess the WCAG compliance of
+        their work.
+      </NxP>
+      <NxP>
+        WCAG 2.1 Verification - 116 Questions | Cumulative list
```

Comment:
I'm a little unclear on what this line is really supposed to be.  It's a description of the rest of the page I guess, but being so terse, it feels like maybe it should be a header? Or alternatively it could be kept as a paragraph but expanded to say the same thing in full sentences (e.g. "The checklist below represents 116 questions that provide a relatively quick way to assess WCAG 2.1 compliance...").

Additionally I'm not sure about having the 116 number.  That'll be prone to getting out of sync with the list itself, and also probably isn't particularly valuable.  I'd be in favor of removing that part.

Reply:
I am fine with removing this line. It was more relevant when I was working between four documents -- Level A, Level AA, Level AAA and cumulative iterations of this list.

Reply:
> I am fine with removing this line. It was more relevant when I was working between four documents -- Level A, Level AA, Level AAA and cumulative iterations of this list.

Remover per above: https://github.com/sonatype/sonatype-react-shared-components/pull/657/commits/4caf771c01a7ef53b64c29e3ff2cd42ef1fc1fe9